core/components/atoms/slider/__tests__/Slider.test.tsx

```diff
@@ -40,3 +40,16 @@ describe('Slider component', () => {
 
   testHelper(mapper, testFunc);
 });
+
+describe('slider component', () => {
+  it('check for onChange function call', () => {
+    const onChangeHandler = jest.fn();
+    const { getByTestId } = render(<Slider label="Slider Label" onChange={onChangeHandler} />);
+    const sliderHandle = getByTestId('DesignSystem-MultiSlider-Handle');
+    fireEvent.click(sliderHandle);
+    fireEvent.keyUp(sliderHandle, { keyCode: 37 });
+    fireEvent.keyDown(sliderHandle, { keyCode: 37 });
+    fireEvent.keyDown(sliderHandle, { keyCode: 39 });
+    expect(onChangeHandler).toBeCalled();
+  });
```

Comment:
Please add a test case to check if correct value is passed in `onChangeHandler` callback function